tests/test_project.py

```diff
@@ -614,18 +598,16 @@ def test_repair_corrupted_workspace(self):
                 for job in self.project:
                     # Accessing the job state point triggers validation of the
                     # state point manifest file
-                    job.statepoint
-
-            with pytest.raises(JobsCorruptedError):
-                self.project.repair()
+                    job.statepoint()
```

Comment:
I assume the call is needed due to lazy loading?

Reply:
Yes, I changed this just to be sure that loading and validation are performed. I think this test might have skipped that check on the second (post-repair) iteration until I made this change to fix it. Iterating over the project is not enough, the state point must be accessed.

Reply:
Might be best to than explicitly check the return value, e.g., with:
```suggestion
                    assert len(job.statepoint()) > 0
```
Or something like that.

Reply:
Good idea. Done in d194c3bc.